PerfNext/public/lib/js/util.js

```diff
@@ -1112,7 +1109,7 @@ function submitJob() {
 		console.log('submitJob(): Inside ajaxStart');			
 	});
 	
-	$('#submit-console-output').html('Submitting Jobs to Jenkins. Please wait for Jenkin Build URLs to be generated...');
+	$('#submit-console-output').html('Submitting Jobs to Jenkins. Please wait (up to 5 seconds) for Jenkins Build URLs to be generated...');
```

Comment:
Once we have the Jenkins links, please add an instruction saying that the user needs to look at the Grandparent job's console output to get the parent builds for the baseline and test builds since Perf Compare only supports Parent builds for now?